### PR TITLE
LPS-113429 set "navigation" on action url

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectNotFoundEntriesManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -204,6 +204,14 @@ public String getDefaultEventHandler() {
 		).build();
 	}
 
+	public PortletURL getIteratorURL() {
```
Comment (4lejandrito) [inline modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectNotFoundEntriesManagementToolbarDisplayContext.java:207]: @AliciaGarciaGarcia I think there is a better way to fix this, we shouldn't need to add the navigation parameter. I think the problem is in the `RedirectNotFoundEntriesDisplayContext._getPorteltURL` method that should return a clone of the current URL to include all the parameters, and not only the navigation. 

I am not 100% sure though, we can discuss it tomorrow.

Comment (4lejandrito) [reply modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectNotFoundEntriesManagementToolbarDisplayContext.java:207]: See https://github.com/liferay-lima/liferay-portal/pull/29.

**Pull request closed without merging**